system-addon/test/unit/lib/TelemetryFeed.test.js

```diff
@@ -145,13 +171,20 @@ describe("TelemetryFeed", () => {
         assert.propertyVal(ping, "session_id", sessionID);
         assert.propertyVal(ping, "page", "about:newtab");
       });
+      it("should create an unexpected session ping if no session yet portID is supplied", async () => {
+        const ping = await instance.createPing("foo");
+
+        assert.validate(ping, BasePing);
```

Comment:
This one actually wants to be `BasePing` as we're just creating a ping. I changed the description of the test and separately added to `#createSessionEndEvent` testing with `perf: {load_trigger_type: "unexpected"}` and `assert.validate(ping, SessionPing);`